config-connector/solutions/sql/kpt/postgres-ha/README.md

```diff
@@ -0,0 +1,65 @@
+PostgreSQL High Availability
+==================================================
+# NAME
+  postgres-ha
+# SYNOPSIS
+  Config Connector compatible yaml files to configure a high availability PostgreSQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/sql/kpt/postgres-ha postgres-ha
+  ```
+# SETTERS
+|       NAME        |         VALUE          |     SET BY      |          DESCRIPTION           | COUNT |
+|-------------------|------------------------|-----------------|--------------------------------|-------|
+| database-1-name   | postgres-ha-database-1 | package-default | name of first SQL database     | 1     |
+| database-2-name   | postgres-ha-database-2 | package-default | name of second SQL database    | 1     |
+| external-ip-range | 192.10.10.10/32        | package-default | ip range to allow to connect   | 4     |
+| instance-name     | postgres-ha-solution   | package-default | name of SQL instance           | 9     |
+| password-1        | ${PASSWORD_1?}         | PLACEHOLDER     | password of  user              | 1     |
+| password-2        | ${PASSWORD_2?}         | PLACEHOLDER     | password of  user              | 1     |
+| password-3        | ${PASSWORD_3?}         | PLACEHOLDER     | password of  user              | 1     |
+| region            | us-central1            | package-default | region of SQL instance         | 4     |
+| username-1        | ${USERNAME_1?}         | PLACEHOLDER     | name of  user                  | 1     |
+| username-2        | ${USERNAME_2?}         | PLACEHOLDER     | name of  user                  | 1     |
+| username-3        | ${USERNAME_3?}         | PLACEHOLDER     | name of  user                  | 1     |
+| zone              | us-central1-c          | package-default | zone of main instance          | 1     |
+| zone-backup-1     | us-central1-a          | package-default | zone of first backup instance  | 1     |
```

Comment:
These are not backup instances, they are replication instances. Can you please refer to all field names as `replication` not `backup`.

Reply:
Done